packages/haul-core-legacy/src/compiler/worker/runWebpackCompiler.js

```diff
@@ -89,10 +186,6 @@ module.exports = async function runWebpackCompiler({
 
   const compiler = webpack(apps);
 
-  // As of Webpack 4.12.0, |outputFileSystem| must be set since there is no
-  // fallback despite the documentation stating otherwise.
-  compiler.outputFileSystem = fs;
```

Comment:
was this fixed in some recent version? if so, we need to bump peer dep

Reply:
we don't use custom filesystem, the fs here was actually the leftover from where we were using `memory-fs`, but now it was pointing to `fs` module from Node.